app/models/course/assessment/question/multiple_response.rb

```diff
@@ -45,6 +45,15 @@ def initialize_duplicate(duplicator, other)
     self.options = duplicator.duplicate(other.options)
   end
 
+  # returns the type of question as multiple response or multiple choice
+  def type
+    if options.where(correct: true).count > 1
```

Comment:
Should do the check using `#multiple_choice?` here.

Reply:
I tried using multiple choice instead but the tests kept failing as `#multiple_choice?` is returning `true` even when more than one option is correct.